xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/internal/job/MoveJob.java

```diff
@@ -223,6 +226,19 @@ private void move(DocumentReference oldReference, DocumentReference newReference
             this.progressManager.popLevelProgress(this);
         }
     }
+    
+    private ObservationManager getObservationManager()
```

Comment:
Why this method ?

Reply:
Is this not needed?

Reply:
You're injecting it already.... with `@Inject`.... so it's definitely not needed